docs/visual-basic/language-reference/operators/isnot-operator.md

```diff
@@ -18,26 +18,46 @@ result = object1 IsNot object2
 ```
 
 ## Parts
- `result`
- Required. A `Boolean` value.
 
- `object1`
- Required. Any `Object` variable or expression.
+- `result`
 
- `object2`
- Required. Any `Object` variable or expression.
+  Required. A `Boolean` value.
+
+- `object1`
+
+  Required. Any `Object` variable or expression.
+
+- `object2`
+
+  Required. Any `Object` variable or expression.
 
 ## Remarks
- The `IsNot` operator determines if two object references refer to different objects. However, it does not perform value comparisons. If `object1` and `object2` both refer to the exact same object instance, `result` is `False`; if they do not, `result` is `True`.
 
- `IsNot` is the opposite of the `Is` operator. The advantage of `IsNot` is that you can avoid awkward syntax with `Not` and `Is`, which can be difficult to read.
+The `IsNot` operator determines if two object references refer to different objects. However, it does not perform value comparisons. If `object1` and `object2` both refer to the exact same object instance, `result` is `False`; if they do not, `result` is `True`.
+
+`IsNot` is the opposite of the `Is` operator. The advantage of `IsNot` is that you can avoid awkward syntax with `Not` and `Is`, which can be difficult to read.
 
  You can use the `Is` and `IsNot` operators to test both early-bound and late-bound objects.
 
 ## Example
- The following code example uses both the `Is` operator and the `IsNot` operator to accomplish the same comparison.
 
- [!code-vb[VbVbalrOperators#29](~/samples/snippets/visualbasic/VS_Snippets_VBCSharp/VbVbalrOperators/VB/Class1.vb#29)]
+The following code example uses both the `Is` operator and the `IsNot` operator to accomplish the same comparison.
+
+[!code-vb[VbVbalrOperators#29](~/samples/snippets/visualbasic/VS_Snippets_VBCSharp/VbVbalrOperators/VB/Class1.vb#29)]
+
+## Use TypeOf operator with IsNot operator
+
+Starting with Visual Basic 14, you can use the `TypeOf` operator with the `IsNot` operator. For example, instead of writing:
+
+```vb
+If Not TypeOf sender Is Button Then
+```
+
+You can use:
+
+```vb
+If TypeOf sender IsNot Button Then
+```
```

Comment:
I think we can remove the "old way" and just simply say you can use it with the type of and show it. The previous section brings out that IsNot is better than Is and Not.

Reply:
Also, I think we should make a similar update to the `Is operator` article. That article does bring out that it can be used with `TypeOf`, but it's a note. I think it's nice to have it as a section, like you did here.